CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.2
+* Enabled to sign PUT methods
+* Added an example of presigning a URL to upload files to an S3 bucket (or DigitalOcean Space)
```

Comment:
Having additional examples is always nice (thanks for posting that!) but I'm not sure it should be mentioned in the changelog (considering we've got another patch specified).